ACPs/TEMPLATE.md

```diff
@@ -1,13 +1,11 @@
-```text
-ACP: <PR Number>
-Title: <ACP title>
-Author(s): <a list of the author's name(s) and optionally contact info: FirstName LastName <foo@bar.com>>
-Discussions-To: <GitHub Discussion URL (POPULATED BY MAINTAINER, DO NOT SET)>
-Status: <Proposed, Implementable, Activated, Stale>
-Track: <Standards, Best Practices, Meta, Subnet>
-Replaces (*optional): <ACP number>
-Superseded-By (*optional): <ACP number>
-```
+| ACP | PR Number |
+| :--- | :--- |
+| **Title** | ACP title |
+| **Author(s)** | A list of the author's name(s) and optionally contact info: FirstName LastName ([@GitHubUsername](./README.md) or [email@address.com](./README.md)) |
+| **Status** | Proposed, Implementable, Activated, Stale ([Discussion](./README.md)) |
+| **Track** | Standards, Best Practices, Meta, Subnet |
+| **Replaces (\*optional)** | [ACP-XX](./README.md) |
+| **Superseded-By (\*optional)** | [ACP-XX](./README.md) |
```

Comment:
@michaelkaplan13 what do you think about that:
```suggestion
| **Author(s)** | A list of the author's name(s) and optionally contact info: FirstName LastName ([@GitHubUsername](https://github.com/Username) or [email@address.com](mailto:email@address.com)) |
| **Status** | Proposed, Implementable, Activated, Stale ([Discussion](https://github.com/avalanche-foundation/ACPs/discussions/XXX)) |
| **Track** | Standards, Best Practices, Meta, Subnet |
| **Replaces (\*optional)** | [ACP-XX](https://github.com/avalanche-foundation/ACPs/blob/main/ACPs/XXX/README.md) |
| **Superseded-By (\*optional)** | [ACP-XX](https://github.com/avalanche-foundation/ACPs/blob/main/ACPs/XXX/README.md) |
```

Reply:
Makes sense to me 👍